.github/workflows/release-version.yml

```diff
@@ -45,19 +45,29 @@ jobs:
       - name: Install dependencies
         run: yarn install
       - name: Bump CI integration version
-        # The `version` command only bumps the version in package.json. It doesn't commit nor tag.
-        run: yarn version ${{ github.event.inputs.semver }}
+        id: bump-version
+        run: |
+          OLD_VERSION_TAG="v$(jq -r '.version' package.json)" # e.g. v1.0.0
+
+          # The `version` command only bumps the version in package.json. It doesn't commit nor tag.
+          yarn version ${{ github.event.inputs.semver }}
+
+          NEW_VERSION_TAG="v$(jq -r '.version' package.json)" # e.g. v1.1.0
+
+          # Update README with new version
+          sed -i -E "s/$OLD_VERSION_TAG/$NEW_VERSION_TAG/" README.md
```

Comment:
Do we want the `-E` here? I noticed you didn't have it in https://github.com/DataDog/synthetics-test-automation-circleci-orb/pull/38